README.md

```diff
@@ -4,7 +4,7 @@ MaxMind::DB::Reader::XS - Fast XS implementation of MaxMind DB reader
 
 # VERSION
 
-version 1.000008
+version 1.000009
```

Comment:
`1.000010`

Reply:
`Dist::Zilla` automatically generates these on a release. When we release `1.000010`, this file will be updated.